static/app/views/dashboardsV2/widgetCard/chart.tsx

```diff
@@ -417,13 +417,20 @@ class WidgetCardChart extends Component<WidgetCardChartProps, State> {
       yAxis: {
         axisLabel: {
           color: theme.chartLabel,
-          formatter: (value: number) =>
-            timeseriesResultsTypes
-              ? axisLabelFormatterUsingAggregateOutputType(
-                  value,
-                  timeseriesResultsTypes[axisLabel]
-                )
-              : axisLabelFormatter(value, aggregateOutputType(axisLabel)),
+          formatter: (value: number) => {
+            if (timeseriesResultsTypes) {
+              const outputType = Object.values(timeseriesResultsTypes).reduce(
+                (previousType, currentType, index) => {
+                  if (index > 0 && currentType !== previousType) {
+                    return 'number';
+                  }
+                  return currentType;
+                }
+              );
```

Comment:
Can we do something here with sets, using the knowledge that we expect a single type?

```js
const outputType = new Set(Object.values(timeseriesResultsTypes)).size === 1 ? timeseriesResultTypes[axisLabel] : 'number';
```
? I think it's a bit clearer

Reply:
👍 done!